Sources/PackageDescription/PackageDescription.swift

```diff
@@ -27,50 +27,61 @@ import Foundation
-///         ]
-///     )
+/// let package = Package(
+///     name: "MyLibrary",
+///     platforms: [
+///         .macOS(.v10_15),
+///     ],
+///     products: [
+///         .library(name: "MyLibrary", targets: ["MyLibrary"])
+///     ],
+///     dependencies: [
+///         .package(url: "https://url/of/another/package/named/utility", from: "1.0.0")
+///     ],
+///     targets: [
+///         .target(name: "MyLibrary", dependencies: ["Utility"]),
+///         .testTarget(name: "MyLibraryTests", dependencies: ["MyLibrary"])
+///     ]
+/// )
+/// ```
 ///
-/// The package manifest must begin with the string `// swift-tools-version:`,
-/// followed by a version number such as `// swift-tools-version:5.3`.
+/// In Swift tools versions earlier than 5.4, the package manifest must begin with the string `// swift-tools-version:`
+/// followed by a version number specifier. Version 5.4 and later has relaxed the whitespace requirements.
+/// The following code listing shows a few examples of valid declarations of the Swift tools version:
 ///
-/// The Swift tools version declares:
+/// ```swift
+/// // swift-tools-version:3.0.2
+/// // swift-tools-version:3.1
+/// // swift-tools-version:4.0
+/// // swift-tools-version:5.3
+/// // swift-tools-version: 5.6
+/// ```
 ///
-///   - The version of the PackageDescription framework
-///   - The Swift language compatibility version to process the manifest
-///   - The required minimum version of the Swift tools to use the package
-///
-/// Each version of Swift can introduce updates to the PackageDescription
-/// library, but the previous API version is available to packages that declare
-/// a prior tools version. This behavior allows you take advantage of new
-/// releases of Swift, the Swift tools, and the PackageDescription framework,
-/// without having to update your package manifest and without losing access to
-/// existing packages.
+/// The Swift tools version declares the version of the `PackageDescription`
+/// library, the minimum version of the Swift tools and Swift language
+/// compatibility version to process the manifest, and the required minimum
+/// version of the Swift tools to use the Swift package. Each version of Swift
+/// can introduce updates to the PackageDescription framework, but the previous
+/// API version is available to packages which declare a prior tools version.
+/// This behavior means you can take advantage of new releases of Swift, the Swift
+/// tools, and the PackageDescription library, without having to update your
+/// package's manifest or losing access to existing packages.
 public final class Package {
     /// The name of the Swift package.
+    ///
+    /// If the name of the package is `nil`, Swift Package Manager deduces the name of the
+    /// package using its Git URL.
     public var name: String
 
-    /// The list of supported platforms with a custom deployment target.
+    /// The list of platforms supported by the package.
```

Comment:
This isn't exactly what this property represents, unfortunately ("unfortunately" because the name suggests that it is).  What this does is specify the minimum version required for each of the platforms, e.g. macOS 10.15 for macOS, etc.  So I suggest "The list of minimum versions for supported platforms".